GimmeDatAPI.PlainOldClrObjects/MenuPrices.cs

```diff
@@ -0,0 +1,9 @@
+namespace GimmeDatAPI.PlainOldClrObjects
+{
+    public class MenuPrices
+    {
+        public string SoupPrice { get; set; }
+        public string StandardMealPrice { get; set; }
+        public string DeluxeMealPrice { get; set; }
```

Comment:
Czy na pewno chcemy trzymać wszystkie ceny jako stringi?

Reply:
Też się nad tym wahałem. 

Bo z jednej strony jeśli chodzi operacje na pieniądzach to oczywistym wydaje się decimal żeby nie gubić precyzji.
Ale z drugiej to klasa MenuPrices wystawiona jest bezpośrednio w kontrolerze jako JSON. A z nim to wiadomo że czasem nie wiadomo: [stackoverflow](https://stackoverflow.com/questions/35709595/why-would-you-use-a-string-in-json-to-represent-a-decimal-number).

Możliwą funkcjonalnością jaka wpadła mi to głowy dzięki Twojemu pytaniu jest sprawdzanie czy konwersja tego stringa (otrzymanego ze strony) na decimal się powiedzie i dodanie dodatkowego pola (lub pól) przechowującego wynik tej konwersji - tak aby druga strona wiedziała czy string z ceną ma jakiś sens czy może poszło coś nie tak przy scrape'owaniu. Co myślisz?